qiskit/__init__.py

```diff
@@ -127,3 +126,23 @@ def __getattr__(self, attr):
 
 Aer = AerWrapper()
 IBMQ = IBMQWrapper()
+
+__all__ = [
+    "Aer",
+    "AncillaRegister",
+    "BasicAer",
+    "ClassicalRegister",
+    "_config",
+    "IBMQ",
+    "MissingOptionalLibraryError",
+    "QiskitError",
+    "__qiskit_version__",
+    "__version__",
+    "QuantumCircuit",
+    "QuantumRegister",
+    "assemble",
+    "execute",
+    "schedule",
+    "sequence",
+    "transpile",
+]
```

Comment:
A super minor and nitpicky complaint, but for the future, it might be nice to impose an order on these.  (That said, if you remove the underscore ones I commented on, then they _will_ be sorted!)